### DISCUSSION TITLE
Extend Elementor Loop Carousel Widget

### DISCUSSION THREAD

**Discussion by nicolas-pinadata** [elementor/elementor#22836] [Developers Community]
Hi !

I wanted to create a copy of the **Elementor Loop Carousel Widget**. To do that, I extended the widget. All I want is to add a "<div>" at the end of the loop.

The thing is, I created the tabs (Content, Style, Advanced) but they all are empty. I wanted to display all the same configurations as the original Loop Carousel Widget, but they are all empty.

Here is my PHP code : [https://drive.google.com/file/d/1e3AFpi8g0bPZQtxstGFZTFIIHMZqGHls/view?usp=sharing](https://drive.google.com/file/d/1e3AFpi8g0bPZQtxstGFZTFIIHMZqGHls/view?usp=sharing)

Please, help.

**Comment by susanhope** [comment]
Hi @pinadata we only just started building this small community, not many people know about it YET! To help you out, I;ve just shared your request for help in our main (huge) community on Facebook (it is one of the comments in the post). See the post here: https://www.facebook.com/groups/Elementors/posts/6067579743367761/

I hope someone will come along soon to help you out soon.
